src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/StaffRefDataFunctionalTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.reform.cwrdapi;
+
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.cwrdapi.controllers.response.StaffWorkerSkillResponse;
+import uk.gov.hmcts.reform.cwrdapi.util.FeatureToggleConditionExtension;
+import uk.gov.hmcts.reform.cwrdapi.util.ToggleEnable;
+import uk.gov.hmcts.reform.lib.util.serenity5.SerenityTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.cwrdapi.util.FeatureToggleConditionExtension.getToggledOffMessage;
+
+@ComponentScan("uk.gov.hmcts.reform.cwrdapi")
+@WithTags({@WithTag("testType:Functional")})
+@ActiveProfiles("functional")
+@SerenityTest
+@SpringBootTest
+@Slf4j
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class StaffRefDataFunctionalTest extends AuthorizationFunctionalTest {
```

Comment:
and also change the class name to StaffRefDataSkillsFunctionalTest

Reply:
done